src/main/java/telraam/logic/positioner/nostradamus/Nostradamus.java

```diff
@@ -0,0 +1,154 @@
+package telraam.logic.positioner.nostradamus;
+
+import org.jdbi.v3.core.Jdbi;
+import telraam.database.daos.BatonSwitchoverDAO;
+import telraam.database.daos.StationDAO;
+import telraam.database.daos.TeamDAO;
+import telraam.database.models.BatonSwitchover;
+import telraam.database.models.Detection;
+import telraam.database.models.Station;
+import telraam.database.models.Team;
+import telraam.logic.positioner.Position;
+import telraam.logic.positioner.PositionSender;
+import telraam.logic.positioner.Positioner;
+
+import java.util.*;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+public class Nostradamus implements Positioner {
+    private static final Logger logger = Logger.getLogger(Nostradamus.class.getName());
+    private final int INTERVAL_CALCULATE_MS = 500; // How often to handle new detections (in milliseconds)
+    private final int INTERVAL_FETCH_MS = 60000; // Interval between fetching baton switchovers (in milliseconds)
+    private final int INTERVAL_DETECTIONS_MS = 3000; // Amount of milliseconds to group detections by
+    private final int MAX_NO_DATA_MS = 30000; // Send a stationary position after receiving no station update for x amount of milliseconds
+    private final int AVERAGE_AMOUNT = 10; // Calculate the median running speed of the last x intervals
```

Comment:
median or average?

Reply:
It's the only reference left to average. All the 10 other references say median so it's median but changed it anyway [877fe72](https://github.com/12urenloop/Telraam/pull/142/commits/877fe72e94662b1b303f2f14819238420283cf84)